### DISCUSSION TITLE
How to set a scroll point then move the cursor to scroll?

### DISCUSSION THREAD

**Discussion by maxxracc13** [artginzburg/MiddleClick#106] [Q&A]
I downloaded the cask and it seems like nothing happened? The installation completed with seemingly no problems, but no applications opened and middle clicking (fn + click, cmd + click, and triple tap) does nothing. Is it because I'm running monterey?

Edit: Im dumb, it was in my applications and I needed to run it... Although, im having trouble with the functionality of middle click scrolling that i can do with a mouse. Is there a way to use the middle click to set a scroll point then move the cursor to scroll like you can with a mouse?

**Comment by artginzburg** [comment]
Duplicate of #72
